StortSpelProjekt/Sandbox/src/GameEntity.h

```diff
@@ -0,0 +1,17 @@
+#ifndef GAMEENTITY_H
+#define GAMEENTITY_H
+
+#include "Engine.h"
+#include "PlayerInputComponent.h"
+
+// Temporary
+#include "SandboxComponent.h"
+
+class GameEntity : public Entity
+{
+public:
+	GameEntity(std::string entityName);
+	virtual ~GameEntity();
+};
+
+#endif
```

Comment:
In my mind, Entity is GameEntity. Explain the reason to use a GameEntity instead of an Entity